README.md

```diff
@@ -0,0 +1,32 @@
+This repository contains the code and content deployed to alpha.nhs.uk
+
+## Dependencies
+
+  - Ruby 2.2.3
+  - Bundler
```

Comment:
Nice to have a link to `http://bundler.io/` and (maybe) an install hint `gem install bundler`